src/Microsoft.AspNetCore.Cors/Internal/CORSLoggerExtensions.cs

```diff
@@ -64,6 +66,11 @@ static CORSLoggerExtensions()
                 LogLevel.Warning,
                 9,
                 "Failed to apply CORS Response headers.");
+
+            _noCorsPolicyFound = LoggerMessage.Define(
+                LogLevel.Information,
+                new EventId(10, nameof(CorsMiddleware)),
```

Comment:
The event name must *also* be unique amongst names, so using `nameof(CorsMiddleware)` is probably not a good idea.

Reply:
(So either remove the event name, or give it a hard-coded name that would be unique within its namespace.)